react/components/Slider/components/Dots.js

```diff
@@ -7,10 +7,10 @@ import PropTypes from 'prop-types'
  */
 export default class Dots extends Component {
   render() {
-    const { className, style, dots, cssClass } = this.props
+    const { className, style, dots, cssClass, dotsClasses } = this.props
```

Comment:
Maintain conciseness, `cssClasses` or `dotsClass`

Reply:
Ok, i will update to `customClass`